src/main/java/com/codingnomads/betty/data/models/Tweet.java

```diff
@@ -0,0 +1,52 @@
+package com.codingnomads.betty.data.models;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "tweets")
+public class Tweet {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    @Size(max = 15)
+    private String language;
+    @NotBlank
+    private String text;
+    @NotBlank
+    private String createdAt;
```

Comment:
Hey Cristian, just FYI I changed the type for createdAt from String to Date as part of integration :)